src/App.js

```diff
@@ -4,41 +4,46 @@ import $ from 'jquery';
 import logo from './logo.svg';
 
 import './App.scss';
-import './assets/sass/main.scss'
+import './assets/sass/main.scss';
```

Comment:
The rest of these changes are just the result of running prettier. Its recommended to run prettier on all files. While we both may have different styles when writing CSS or really any type of code, in the end, prettier will format them all the same, which will greatly reduce the chance of merge conflicts, and reduce cognitive overhead for devs on the project.